homeassistant/components/shopping_list/services.yaml

```diff
@@ -10,3 +10,13 @@ complete_item:
     name:
       description: The name of the item to mark as completed.
       example: Beer
+uncomplete_item:
+  description: Marks an item as incomplete in the shopping list.
```

Comment:
You've written it here as `incomplete` so makes more sense to match it in the code